pandas/core/series.py

```diff
@@ -4352,43 +4352,6 @@ def aggregate(self, func=None, axis: Axis = 0, *args, **kwargs):
 
     agg = aggregate
 
-    # error: Signature of "any" incompatible with supertype "NDFrame"  [override]
-    @overload  # type: ignore[override]
-    def any(
-        self,
-        *,
-        axis: Axis = ...,
-        bool_only: bool | None = ...,
-        skipna: bool = ...,
-        level: None = ...,
-        **kwargs,
-    ) -> bool:
-        ...
-
-    @overload
-    def any(
-        self,
-        *,
-        axis: Axis = ...,
-        bool_only: bool | None = ...,
-        skipna: bool = ...,
-        level: Level,
-        **kwargs,
-    ) -> Series | bool:
-        ...
-
-    # error: Missing return statement
-    @doc(NDFrame.any, **_shared_doc_kwargs)
-    def any(  # type: ignore[empty-body]
```

Comment:
`NDFrame.any` doesn't seem to have a doc-string.

Reply:
(if possible, it would be great to add overloads to `NDFrame.any` to avoid returning `bool | Series | DataFrame`)

Reply:
it gets re-defined with a docstring in _add_numeric_operations (which is a pattern im leaning towards getting rid of)